diff -uNr -uNr anaconda-11.1.2.195/dispatch.py anaconda-11.1.2.195.frameos/dispatch.py
--- anaconda-11.1.2.195/dispatch.py	2009-08-05 16:32:40.000000000 +0200
+++ anaconda-11.1.2.195.frameos/dispatch.py	2010-06-10 15:25:22.451089148 +0200
@@ -88,8 +88,8 @@
     ("accounts", ),
     ("reposetup", doRepoSetup, ),
     ("basepkgsel", doBasePackageSelect, ),
-    ("tasksel", ),
-    ("group-selection", ),
+    #("tasksel", ),
+    #("group-selection", ),
     ("postselection", doPostSelection, ),
     ("confirminstall", ),
     ("confirmupgrade", ),
diff -uNr -uNr anaconda-11.1.2.195/gui.py anaconda-11.1.2.195.frameos/gui.py
--- anaconda-11.1.2.195/gui.py	2009-08-05 16:32:40.000000000 +0200
+++ anaconda-11.1.2.195.frameos/gui.py	2010-06-10 15:18:41.134081801 +0200
@@ -71,8 +71,8 @@
     "network" : ("network_gui", "NetworkWindow"),
     "timezone" : ("timezone_gui", "TimezoneWindow"),
     "accounts" : ("account_gui", "AccountWindow"),
-    "tasksel": ("task_gui", "TaskWindow"),    
-    "group-selection": ("package_gui", "GroupSelectionWindow"),
+    #"tasksel": ("task_gui", "TaskWindow"),    
+    #"group-selection": ("package_gui", "GroupSelectionWindow"),
     "confirminstall" : ("confirm_gui", "InstallConfirmWindow"),
     "confirmupgrade" : ("confirm_gui", "UpgradeConfirmWindow"),
     "install" : ("progress_gui", "InstallProgressWindow"),
diff -uNr -uNr anaconda-11.1.2.195/text.py anaconda-11.1.2.195.frameos/text.py
--- anaconda-11.1.2.195/text.py	2009-08-05 16:32:40.000000000 +0200
+++ anaconda-11.1.2.195.frameos/text.py	2010-06-10 15:20:27.059075262 +0200
@@ -61,8 +61,8 @@
                                   "HostnameWindow")),
     "timezone" : ("timezone_text", "TimezoneWindow"),
     "accounts" : ("userauth_text", "RootPasswordWindow"),
-    "tasksel": ("task_text", "TaskWindow"),
-    "group-selection": ("grpselect_text", "GroupSelectionWindow"),    
+    #"tasksel": ("task_text", "TaskWindow"),
+    #"group-selection": ("grpselect_text", "GroupSelectionWindow"),    
     "confirminstall" : ("confirm_text", "BeginInstallWindow"),
     "confirmupgrade" : ("confirm_text", "BeginUpgradeWindow"),
     "install" : ("progress_text", "setupForInstall"),
